beluga_amcl/src/amcl_node.cpp

```diff
@@ -922,10 +923,16 @@ void AmclNode::laser_callback(ExecutionPolicy&& exec_policy, sensor_msgs::msg::L
   const auto update_start_time = std::chrono::high_resolution_clock::now();
   const auto filter_updated = particle_filter_->update_filter(
       exec_policy, odom_to_base_transform,
-      utils::make_points_from_laser_scan(
-          *laser_scan, base_to_laser_transform, static_cast<std::size_t>(get_parameter("max_beams").as_int()),
-          static_cast<float>(get_parameter("laser_min_range").as_double()),
-          static_cast<float>(get_parameter("laser_max_range").as_double())),
+      beluga_ros::LaserScan{
+          laser_scan,
+          base_to_laser_transform,
+          static_cast<std::size_t>(get_parameter("max_beams").as_int()),
+          get_parameter("laser_min_range").as_double(),
+          get_parameter("laser_max_range").as_double(),
+      }
+              .points_in_cartesian_coordinates() |
+          ranges::views::transform([](const auto& value) { return std::make_pair(value.x(), value.y()); }) |
```

Comment:
Indentation.

Reply:
This is how the linter wants this... It won't last long once follow-up changes land.